.github/workflows/pr-auto-milestone.yml

```diff
@@ -39,6 +39,32 @@ jobs:
           configuration-path: .github/pr-2-milestone-config.yml@master
           configuration-repo: geoadmin/.github
 
+  check-milestone-branch:
+    needs: add-milestone
+    name: Check milestone branch on open PRs
+    runs-on: ubuntu-latest
+    if: ${{ github.base_ref == 'master' && startsWith(github.head_ref, 'develop-') }}
+    steps:
+      - name: Count and check open PRs on milestone branch
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} 
+          github_repo: ${{ github.repository }}
+          github_head_ref: ${{ github.head_ref }}
+          github_base_ref: ${{ github.base_ref }}
```

Comment:
No need to use the `env` section, you can below in `run` directly use the variables with `${{ VARIABLE }}` for example `${{ github.head_ref }}`. However the `GITHUB_TOKEN` should be kept in the `env` section otherwise the token will be displayed in the log output of the action which would be a security issue. But for the other as they are not sensitive data you can put them in the template.